packages/control-property-editor/test/unit/toolbar/ModeSwitcher.test.tsx

```diff
@@ -38,3 +38,28 @@ test('renders ModeSwitcher', () => {
     editBtn.click();
     expect(dispatch).toBeCalledWith(setAppMode('adaptation'));
 });
+
+test('renders ModeSwitcher with changed button names for ADP', () => {
+    const { dispatch, store } = render(<ModeSwitcher />, { initialState: adpState });
```

Comment:
You don't need to create a copy of an initial state, you can just set the scenario like this `store.dispatch(setProjectScenario('ADAPTATION_PROJECT'));`.

Reply:
Done in [6c93187](https://github.com/SAP/open-ux-tools/pull/2068/commits/6c9318770081a6025469af63e0a5939a2a2ef178)